### DISCUSSION TITLE
How to change default icon for window app?

### DISCUSSION THREAD

**Discussion by QiCodeCN** [flet-dev/flet#585] [Q&A]
Hello, I'm writing a cross-platform desktop application. The packager replaces the icon, but launches the application by default. This creates some confusion for users. Please ask how you can change it.

![132017](https://user-images.githubusercontent.com/52949189/202626581-ab6be06a-a3d4-4064-b90a-c33b17dad656.jpg)

Thanks a lot.

**Comment by keyiflerolsun** [comment]
Duplicate;

https://github.com/flet-dev/flet/discussions/378
